gapis/api/gles/replay.go

```diff
@@ -275,7 +275,7 @@ func (t *destroyResourcesAtEOS) Flush(ctx context.Context, out transform.Writer)
 		// change to the gles.api file.
 
 		// Delete all Renderbuffers.
-		renderbuffers := make([]RenderbufferId, 0, len(c.Objects.Shared.Renderbuffers)-3)
+		renderbuffers := make([]RenderbufferId, 0, len(c.Objects.Shared.Renderbuffers))
```

Comment:
Unrelated fix for a panic. If there's no shared renderbuffers, then you'll get a negative cap.